docs/guides/quick-start/miranum-connect/message-quickstart.md

```diff
@@ -0,0 +1,77 @@
+---
+id: message-quickstart
+title: Correlate a Message
+sidebar_label: Correlate a Message
+description: "This section introduces the concept of correlating a message using the Miranum Connect API."
+---
+
+# Correlating Messages with Miranum Connect
+
+Message correlation is an essential feature of Miranum Connect, enabling your application to interact asynchronously with running workflows. This guide will walk you through how to use the Message Correlation feature effectively, allowing you to update the state of your workflows based on external triggers.
+
+:::info What You Will Learn
+- How to incorporate necessary dependencies to use Miranum Message
+- How to implement and use the Message Correlation feature
+:::
+
+:::note
+This guide assumes familiarity with Java and Maven.
+:::
+
+## Add the Necessary Dependencies
+
+To use the message correlation feature of Miranum Connect, you must add the `message-api` and an engine-specific dependency to your project. Here is an example of the Maven dependency for the `message-api`:
+
+```xml
+<dependency>
+ <groupId>io.miragon.miranum</groupId>
+ <artifactId>message-api</artifactId>
+ <version>0.2.0</version>
```

Comment:
💭 We have a specific version mentioned here but a${miranum.worker.version} below. I think we should unify this. (e.g. with ${miranum.version} )